Sprint-1/interpret/time-format.js

```diff
@@ -12,13 +12,20 @@ console.log(result);
 // For the piece of code above, read the code and then answer the following questions
 
 // a) How many variable declarations are there in this program?
+//  6 variable declarations
 
 // b) How many function calls are there?
+// 1 function
 
 // c) Using documentation, explain what the expression movieLength % 60 represents
+// The remainder (%) operator returns the remainder left over when one operand is divided by a second operand.
+// If movieLength is 8784 seconds, movieLength % 60 returns 24, indicating that 24 seconds remain after calculating full minutes.
 
 // d) Interpret line 4, what does the expression assigned to totalMinutes mean?
+// It calculates the total minutes in movieLength, excluding seconds that do not complete a full minute.
 
 // e) What do you think the variable result represents? Can you think of a better name for this variable?
+// It represent time, in my opinion better name for this variable is "time"
```

Comment:
"7pm", "07:00", and 8784 second could all be considered as time. Could you come up with a better name that can give people a good idea what value is stored in the variable?